<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>系统设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f5f7fa;
    font-family: 'Segoe UI', 'Microsoft YaHei', sans-serif;
}
QGroupBox {
    border: 1px solid #d1d9e6;
    border-radius: 8px;
    margin-top: 1.5em;
    padding: 16px 12px;
    font-weight: 600;
    color: #2d3748;
    background-color: white;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 12px;
    padding: 0 8px;
}
QPushButton {
    background-color: #4a6cf7;
    color: white;
    border-radius: 6px;
    padding: 8px 16px;
    border: none;
    font-weight: 500;
}
QPushButton:hover {
    background-color: #3a5aeb;
}
QPushButton#pushButton_5 {
    background-color: #e2e8f0;
    color: #1e293b;
}
QPushButton#pushButton_5:hover {
    background-color: #cbd5e1;
}
QPushButton#pushButton_6 {
    background-color: #e2e8f0;
    color: #1e293b;
}
QPushButton#pushButton_6:hover {
    background-color: #cbd5e1;
}
QScrollArea {
    border: none;
}
QPlainTextEdit, QLineEdit {
    border: 1px solid #d1d9e6;
    border-radius: 6px;
    padding: 8px 12px;
    background-color: white;
}
QPlainTextEdit:focus, QLineEdit:focus {
    border: 1px solid #93c5fd;
    outline: none;
}
QCheckBox, QRadioButton {
    spacing: 8px;
}
QCheckBox::indicator, QRadioButton::indicator {
    width: 18px;
    height: 18px;
}
QLabel {
    color: #475569;
}

QPlainTextEdit:disabled {
    background-color: #f0f0f0;
    color: #a0a0a0;
    border: 1px solid #d1d9e6;
}

QLabel:disabled {
    color: #a0a0a0;
}

QCheckBox:disabled {
    color: #a0a0a0;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>24</number>
   </property>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="styleSheet">
      <string notr="true">font-size: 20px;
font-weight: bold;
color: #1e293b;
background: transparent;
</string>
     </property>
     <property name="text">
      <string>系统设置</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1135</width>
        <height>1133</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>24</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>微信监听</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="styleSheet">
               <string notr="true">padding: 10px;</string>
              </property>
              <property name="text">
               <string>开启微信监听</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="styleSheet">
               <string notr="true">color: #64748b; 
font-size: 12px;
margin-left: 8px;

background: transparent;
                </string>
              </property>
              <property name="text">
               <string>仅可在微信监听关闭时对下面的项进行设置</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
            <property name="text">
             <string>Python启动参数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">background: transparent;
</string>
            </property>
            <property name="text">
             <string>监听用户列表（使用 ; 分隔）：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="horizontalSpacing">
             <number>24</number>
            </property>
            <property name="verticalSpacing">
             <number>16</number>
            </property>
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBox">
              <property name="styleSheet">
               <string notr="true">background: transparent;</string>
              </property>
              <property name="text">
               <string>登陆时自动启动微信监听</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="checkBox_2">
              <property name="styleSheet">
               <string notr="true">background: transparent;</string>
              </property>
              <property name="text">
               <string>监听时发送相关信息</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="checkBox_3">
              <property name="styleSheet">
               <string notr="true">background: transparent;</string>
              </property>
              <property name="text">
               <string>图片消息监听</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="checkBox_4">
              <property name="styleSheet">
               <string notr="true">background: transparent;</string>
              </property>
              <property name="text">
               <string>文件消息监听</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QCheckBox" name="checkBox_5">
              <property name="styleSheet">
               <string notr="true">background: transparent;</string>
              </property>
              <property name="text">
               <string>语音监听</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>关闭主窗口</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>16</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="styleSheet">
             <string notr="true">background: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>最小化到托盘，不退出程序</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="styleSheet">
             <string notr="true">background: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>退出程序</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>大模型API</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>16</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">background: transparent;</string>
              </property>
              <property name="text">
               <string>硅基流动API（用于图片和文本解析）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">background: transparent;</string>
              </property>
              <property name="text">
               <string>Dify API（用于文档解析）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>提醒设置</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>24</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBox_6">
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
            <property name="text">
             <string>开始前15分钟提醒</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_7">
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
            <property name="text">
             <string>开始时提醒</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>快捷键设置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="horizontalSpacing">
           <number>24</number>
          </property>
          <property name="verticalSpacing">
           <number>16</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
            <property name="text">
             <string>添加事件</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
            <property name="text">
             <string>上一个事件</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="pushButton_4">
            <property name="styleSheet">
             <string notr="true">background-color: #e2e8f0;
color: #1e293b;</string>
            </property>
            <property name="text">
             <string>Page Down</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_7">
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
            <property name="text">
             <string>下一个事件</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_3">
            <property name="styleSheet">
             <string notr="true">background-color: #e2e8f0;
color: #1e293b;</string>
            </property>
            <property name="text">
             <string>Page Up</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="styleSheet">
             <string notr="true">background-color: #e2e8f0;
color: #1e293b;</string>
            </property>
            <property name="text">
             <string>Ctrl+Alt+N</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
